theme-orange/src/main/java/com/orange/ouds/theme/orange/tokens/OrangeColorSemanticTokens.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.theme.orange.tokens
+
+import com.orange.ouds.theme.orange.OrangeTheme
+import com.orange.ouds.theme.tokens.semantic.OudsColorTokenValue
+import com.orange.ouds.theme.tokens.semantic.OudsColorTokens
+
+val OrangeTheme.orangeColorSemanticTokens
```

Comment:
Yes you're right this is here because at a first step I thought that an Orange extended theme must be able to use Orange raw tokens. I forgot to remove it.